### ISSUE TITLE
[v2] Timepicker

### ISSUE THREAD

**Issue by danish1023** [framework7io/Framework7#1871]
@nolimits4web 
Timepicker is very important component and it is still missing. Please add this in v2.
![clockdemo](https://user-images.githubusercontent.com/6543485/31440637-34294fe4-aeae-11e7-87ad-6b2b293216ac.gif)

**Comment by J05HI** [comment]
@danish1023
Yeah amazing idea!!!
We need that :)

**Comment by D-Marc1** [comment]
I mentioned this in a different issue #1686, but I also think simple pickers with date/time related stuff would be useful too, like [here](https://ionicframework.com/docs/api/components/datetime/DateTime/), in addition to this.

**Comment by J05HI** [comment]
@D-Marc1
Yeah hope @nolimits4web will integrate a good date/time component :)

The ability to add it to an input field would be amazing too.

**Comment by cyberbeat** [comment]
When implementing such a time picker like in the animated gif, please don't forget to offer a 24h variant, because in many country "am/pm" are unknown, see https://vuetifyjs.com/components/pickers 24h format, which is done right.

**Comment by J05HI** [comment]
@nolimits4web
This is a highly needed feature!
I would love your if you could add it 😍

**Comment by MiguelQueiroz** [comment]
There are some, using material design.
But i agree simpler ui is also nice.

**Comment by brian-g** [comment]
With the advent of iOS 14, would be nice to also replicate the new iOS time picker. 

![IMG_102657179A50-1](https://user-images.githubusercontent.com/5779252/99557758-a5ddce00-2988-11eb-96ee-3340339cf7b1.jpeg)

**Comment by nolimits4web** [comment]
This issue was incorrectly closed on 2026-05-19 by a compromised credential (Shai-Hulud npm supply-chain worm). Reopening — apologies for the noise.
